tox.ini

```diff
@@ -159,3 +159,28 @@ commands_post =
       -e @provisioner/tests/vars.yml \
       -e @provisioner/tests/ci-common.yml \
       -e @provisioner/tests/ci-networking.yml
+
+[testenv:workshop-smart_mgmt]
+sitepackages =
+    {[testenv:workshops]sitepackages}
+whitelist_externals =
+    {[testenv:workshops]whitelist_externals}
+deps =
+    {[testenv:workshops]deps}
+setenv =
+    {[testenv:workshops]setenv}
+passenv =
+    {[testenv:workshops]passenv}
+commands_pre =
+    {[testenv:workshops]commands_pre}
+commands =
+    ansible-playbook provisioner/provision_lab.yml \
+      -e @provisioner/tests/vars.yml \
+      -e @provisioner/tests/ci-common.yml \
+      -e @provisioner/tests/ci-smart_mgmt.yml
+    bash ./provisioner/tests/verify-rhel.sh
```

Comment:
This is probably not what you want ... You might want to create your own verify script ?

Reply:
I left that intentionally because it works for both labs. Ultimately I'll probably replace it once I figure out better things to verify but for now it's good enough.

Reply:
Ack - Thanks for clarifying. I thought it might have been a copy/paste leftover